docs/architecture-3.0.md

```diff
@@ -13,11 +13,16 @@ The following architecture document addresses
   principles which got lost over the last years due to feature creep
   * making IntelMQ 3.x interoperable with CERT.pl's n6 system
 
+## Implementation evaluation
+The mayor changes listed in this document must go through an evaluation before the final implementation and integration into IntelMQ:
```

Comment:
still have a slight issue with this, could we leave it out of this document for now and let the arch. board (in future) decide how the process should be? I think that's better.